### PR TITLE
Add Azure icons paths

### PR DESCRIPTION
**Pull request by kerak19**

### CODE REVIEW

**Comment by kerak19** [comment]
There's a lot of unrelated changes for other providers - should I ignore them and only have azure related changes?

File: assets/azurerm/icons.json [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+ "azurerm_app_service_custom_hostname_binding": "",
+ "azurerm_app_service_environment": "",
+ "azurerm_app_service_hybrid_connection": "",
+ "azurerm_app_service_plan": "",
+ "azurerm_app_service_slot": "",
+ "azurerm_app_service_slot_virtual_network_swift_connection": "",
+ "azurerm_app_service_source_control_token": "",
+ "azurerm_app_service_virtual_network_swift_connection": "",
+ "azurerm_application_gateway": "Other/10076-icon-service-Application-Gateways.svg",
+ "azurerm_application_insights": "Management + Governance/00012-icon-service-Application-Insights.svg",
```
Comment (xescugc) [inline assets/azurerm/icons.json]: Format the names to not have spaces or characters. so basically `Management_and_Governance`

File: assets/azurerm/icons.json [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+ "azurerm_relay_hybrid_connection": "",
+ "azurerm_relay_namespace": "",
+ "azurerm_resource_group": "",
+ "azurerm_resources": "",
+ "azurerm_role_assignment": "",
+ "azurerm_role_definition": "",
+ "azurerm_route": "",
+ "azurerm_route_filter": "Networking/10071-icon-service-Route-Filters.svg",
+ "azurerm_route_table": "",
+ "azurerm_search_service": "App Services/10044-icon-service-Search-Services.svg",
```
Comment (xescugc) [inline assets/azurerm/icons.json]: `App_Service`.

**Review by xescugc — CHANGES_REQUESTED** [review]
For now just this.

**Comment by kerak19** [comment]
Done

**Review by xescugc — COMMENTED** [review]
RS

**Comment by kerak19** [comment]
Done

**Review by xescugc — APPROVED** [review]

**Pull request merged**